### ISSUE TITLE
Respond to bug report : SV's missing from msk portal

### ISSUE THREAD

**Issue by sheridancbio** [knowledgesystems/pipelines-scrum#941]
## Done Condition (What do we need? Why do we need it? Keep this is small as possible!)
This has been looked into and responded to. (message ino)

**Comment by sheridancbio** [comment]
Determined that reported issue was correct ... we have studies in the msk portal with structural variant profiles that have stable ids such as
- bladder_msk_2022
- egc_msk_tp53_ccr_2022_fusion
- mixed_kunga_msk_2022_fusion

Anusha is looking into why the profile names include _fusion ... there are 304 such cases. Re-imports are being attempted.
I have looked also into the report that these profiles are not visible in the portal study view (which could be a frontend reliance on naming conventions for profiles) but also that the API will not deliver SV events for profiles named like that. When looking for test cases in any of these three profiles with the wrong naming pattern:
bladder_msk_2022_fusion
egc_msk_tp53_ccr_2022_fusion
mixed_kunga_msk_2022_fusion
I could not find a single event in the structural variant table connected to the profile ids above. So I believe that we failed to import SV data for these studies.
